modules/renter/files.go

```diff
@@ -25,13 +25,14 @@ var (
 // contract covers many pieces.
 type file struct {
 	name        string
-	size        uint64
+	size        uint64 // Static - can be accessed without lock.
 	contracts   map[types.FileContractID]fileContract
-	masterKey   crypto.TwofishKey
-	erasureCode modules.ErasureCoder
-	pieceSize   uint64
-	mode        uint32 // actually an os.FileMode
-	mu          sync.RWMutex
+	masterKey   crypto.TwofishKey    // Static - can be accessed without lock.
+	erasureCode modules.ErasureCoder // Static - can be accessed without lock.
+	pieceSize   uint64               // Static - can be accessed without lock.
+	mode        uint32               // actually an os.FileMode
```

Comment:
I'm 95% sure that I did not change any of the persist stuff for the renter. But maybe I missed something on accident.